terraform/hcl/applicable/req_sw_terraform_aws_cloudfront_tls_version.tf

```diff
@@ -0,0 +1,23 @@
+resource "aws_cloudfront_distribution" "vulnerable_example" {
+  default_cache_behavior {
+    allowed_methods  = ["dummy"]
+    cached_methods  = ["dummy"]
+    target_origin_id = "dummy"
+    viewer_protocol_policy = "https-only"
+  }
+
+  viewer_certificate {
+    minimum_protocol_version  = "TLSv1"
+  }
+
+  enabled             = true
+  origin {
+    origin_id = "dummy"
+    domain_name = "dummy"
+  }
+  restrictions {
+    geo_restriction {
+        restriction_type = "none"
+    }
+  }
+}
```

Comment:
[comment]: <> (FrogbotReviewComment)


## 🛠️ Infrastructure as Code Vulnerability
<div align='center'>

| Severity                | Finding                  |
| :---------------------: | :-----------------------------------: |
| ![](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/applicableLowSeverity.png)<br>     Low | Deprecated TLS version was detected |

</div>

<details>
<summary> <b>Full description</b> </summary>
<br>

Resource `aws_cloudfront_distribution` should have `viewer_certificate.minimum_protocol_version = 'TLSv1.2*'` or `viewer_certificate.minimum_protocol_version = 'TLSv1.3*'`

Vulnerable example -
```
resource "aws_cloudfront_distribution" "vulnerable_example" {
  viewer_certificate {
    minimum_protocol_version  = "TLSv1"
  }
}
```

Secure example -
```
resource "aws_cloudfront_distribution" "secure_example" {
  viewer_certificate {
    minimum_protocol_version  = "TLSv1.2_2019"
  }
}

```

</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>